doc/modules/lda_qda.rst

```diff
@@ -187,13 +187,25 @@ an estimate for the covariance matrix). Setting this parameter to a value
 between these two extrema will estimate a shrunk version of the covariance
 matrix.
 
+The Ledoit and Wolf estimator of covariance may not always be the best choice.
+For example if the data are Gaussian distributed the
```

Comment:
By normally distributed, do we mean conditionally on `y` or not?
Because if that's the case, it could be worth repeating that LDA does assume that p(X/y) is gaussian. I.e. OSA is a better choice as long as the original assumptions are verified

Reply:
Yes I mean conditionally to y